### PR TITLE
Normalize $CARGO_HOME in license file path

### PR DESCRIPTION
**Pull request by Skgland**
An attempt at resolving issue #8

### CODE REVIEW

**Comment by Skgland** [comment]
Regarding the remaining clippy warning I am not sure whether `LicenseAndText` should derive `Eq` or if the lint should be allowed for that type.

File: src/found_license.rs [hunk 1]
```diff
@@ -258,8 +258,18 @@ impl Licensed for Package {
             .as_ref()
             .and_then(|license| license.parse::<License>().ok())
             .or_else(|| {
-                self.license_file()
-                    .map(|p| License::File(p.into_std_path_buf()))
+                self.license_file().map(|license_file| {
+                    let license_file = if let Some(stripped_license_file) = home::cargo_home()
```
Comment (sstadick) [inline src/found_license.rs:262]:
```suggestion
                self.license_file().map(|license_file| {
                    // If the license file starts with the cargo_home path, strip it and replace with the ENV var $CARGO_HOME.
                    // This makes licenses comparable across machines. 
                    let license_file = if let Some(stripped_license_file) = home::cargo_home()
```

**Review by sstadick — APPROVED** [review]
@Skgland - thanks for making this! I got part way towards this a while ago and got distracted.

This LGTM. I made one suggestion. I'd also say that we might as well disable the Eq lint if you don't mind doing that. I need to add a `rust-toolchain.toml` for this to avoid regressions like that.

Once those two things are addressed I'll merge and make a release.

File: src/found_license.rs (hunk 1 above)
Comment (Skgland) [reply src/found_license.rs]: I have amended the first commit with the suggested comment.
Did you mean to disable the clippy lint just for this instance or in general?
I have currently done the former in the last commit.

Comment (sstadick) [reply src/found_license.rs]: You did it as I was thinking 👍

**Pull request merged**